package.json

```diff
@@ -30,6 +30,7 @@
     "react-bootstrap": "~0.18.0",
     "react-calendar": "^0.4.0",
     "react-router": "~0.13.2",
+    "react-scroll-components": "^0.2.2",
```

Comment:
I completely forgot about this package (even have it bookmarked). Could this be used for the exercise builder too @dak?